src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.h

```diff
@@ -1907,6 +1907,9 @@ namespace UiaOperationAbstraction
             return false;
         }
 
+        UiaTuple(UiaTuple&&) = default;

```

Comment:
[Rule of Five](https://en.wikipedia.org/wiki/Rule_of_three_(C%2B%2B_programming)#Rule_of_Five), you should probably define a copy assignment operator and destructor as well. Also might be good to put them all next to each other.

Reply:
I noticed that `UiaTypeBase` doesn't have a destructor but has the other four. Shall we add that?

Reply:
Hmm, yeah let's add that as part of this PR.